### PR TITLE
Views

### PR DESCRIPTION
**Pull request by DilsherB**
### Gemfile.rb:
1. gem 'sassc-rails' added and bundle install run.

### Controllers:
1. functions written for index and show page of posts in 'app/controllers/posts_controller.rb'
2. functions written for index and show page of users in 'app/controllers/users_controller.rb'

### Views:
1. logics and html written to show the data in index page of users 'app/views/users/index.html.erb'
2. logics and html written to show the data in show page of users 'app/views/users/show.html.erb'
3. logics and html written to show the data in index page of posts 'app/views/posts/index.html.erb'
4. logics and html written to show the data in show page of posts 'app/views/posts/index.html.erb'

### Extra(Navigation):
1. Navigation between pages defined and implemented at top of every page.
2. At the bottom of every page link of my `github` profile provided.

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,28 @@
-<h1>All users</h1>
\ No newline at end of file
+<h1>All users</h1>
+
+<ul class="users">
+  <% @users.each do |user| %>
+    <li class="flex">
+      <div class="borderb">
+        <img src="<%= asset_path("user-placeholder.png") %>" alt="<%= user.name%>" width=100px >
+      </div>
```
Comment (AmaduKamara) [inline app/views/users/index.html.erb]: - [x] According to the project requirement, you're required to render the user's photo fetched for a particular. Kindly display 3 data for a user, `user.photo, user.name, and user.posts_counter`. Currently, you're displaying a static image from the assets. Kindly work on this 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @DilsherB,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ Good descriptive PR 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [ ] When the app loads, I only see an empty screen with the link to your GitHub account. I created users that are expected to be shown on the screen but nothing shows. Kindly work on that 👍🏽

Screenshot 2023-08-24 at 2 39 49 PM

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: app/views/users/index.html.erb (hunk 1 above)
Comment (DilsherB) [reply app/views/users/index.html.erb]: I have worked on it a lot but still struggling to be successful. I have seen other students' projects who have got approval with the same avatar. All code reviewers' marked this as optional requirement. So please let me submit the project.

Comment (DilsherB) [reply app/views/users/index.html.erb]: https://github.com/DilsherB/ror-blog-app/assets/41000844/bf00bb55-f78e-42aa-8e43-590647284a86

**Comment by ahmetbozaci** [comment]
### Hello @DilsherB 
### _This is an additional review as a result of the second opinion request._ 

- The app is functioning as expected.
image
- [ ] However, the previous code reviewer pointed out that according to the project requirements, you are expected to dynamically render the user's photo fetched from the database. It's important to note that displaying a static image from the assets folder doesn't align with this requirement. To fulfill the objective and make the user's photo meaningful, it must be sourced from the database. Please make the necessary adjustments to address this issue.

_Please remember that even if another student successfully passed with a similar approach, this doesn't guarantee your own success. Each project is evaluated independently based on the requirements, and previous oversights by code reviewers shouldn't be taken as a precedent. Your adherence to the current project requirements is crucial._

___


### Cheers and Happy coding! 💻 👏
____
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @DilsherB, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Good UI design :white_check_mark: 
- [x] All views created correctly :white_check_mark: 
- [x] Application is functioning as expected :white_check_mark: 
- [x] Git flow is used correctly :white_check_mark:  
- [x] Pull request has a descriptive summary :white_check_mark: 
- [x] Implemented all the changes from the previous reviewer :white_check_mark:   

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**